modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -659,6 +670,25 @@ user:
 
 The parser would assume that `user` was the name of the example and that the object had two properties, not one complex property.
 
+If a data type includes a `value` facet and your NamedExample for the type does not include the type's name, the parser assumes that `value` is the name of the type, as in this example:
+
+----
+#%RAML 1.0 NamedExample
+value:
+   a: "b"
+   b: 1
+----
+
+To resolve the warning, simply add the name of the type:
+
```

Comment:
In this case (value key) there won't be a warning, because amf will parse value as a valid name. We suggest to put a name anyway to avoid confusions, but the Api would be ok in both cases.

Reply:
See the latest update for my resolution.